Engine/Plugins/Marketplace/RuntimeMeshImportExport/Source/RuntimeMeshImportExport/Private/AssimpCustom.cpp

```diff
@@ -420,7 +429,20 @@ void FAssimpNode::CreateAssimpMeshesFromMeshData(FAssimpScene& scene, const FRun
 						material->AddProperty(&normalsTextureRelativePath, AI_MATKEY_TEXTURE(aiTextureType_NORMALS, 0));
 	                }
                 }
-            	// Set the opacity of the material (only if it is not 1.0, which means it  is fully opaque)
+				// Set metallicRoughness map
+				{
+					if (!section.materialToExport.gltfMetallicRoughnessTextureRelativePath.IsEmpty())
+					{
+						const aiString gltfMetallicRoughnessTextureRelativePath(TCHAR_TO_UTF8(*section.materialToExport.gltfMetallicRoughnessTextureRelativePath));
+						material->AddProperty(&gltfMetallicRoughnessTextureRelativePath, AI_MATKEY_TEXTURE(aiTextureType_UNKNOWN, 0));
```

Comment:
So you never made it work with the GLTF texture key instead of the `aiTextureType_UNKNOWN`?

Reply:
TBH I didn't continue looking as things already took long enough.

Reply:
Can you try one last time to make it work with the other MACRO? I just think it would be more readable and perhaps work better if we ever change assimp version.